freqtrade/rpc/rpc.py

```diff
@@ -75,6 +76,7 @@ def rpc_trade_status(self) -> Tuple[bool, Any]:
                               current_rate=current_rate,
                               amount=round(trade.amount, 8),
                               close_profit=fmt_close_profit,
+                              stake_value=round(current_rate * trade.amount,6),
```

Comment:
Honestly, i would round to 8 instead of 6 simply to have it aligned to the rest of this output

Reply:
ok let me do this quick